components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -78,30 +125,45 @@ service Policies {
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}";
     option (chef.automate.api.iam.policy).action = "iam:policies:update";
     };
+  /*
+  Get IAM version
+
+  Returns the major and minor version of IAM that your automate installation is running.
+  */
   rpc GetPolicyVersion (GetPolicyVersionReq) returns (GetPolicyVersionResp) {
```

Comment:
I'm not sure where `GetPolicyVersionResp` is defined but it needs a `.` at the end of the line for the `version` field.